### PR TITLE
Create Popup Window with javascript.

### PR DESCRIPTION
**Pull request by Salarhabibi-amir**
First I create a Popup window with using html,css and javascript. When you click on any See Project button it will preview the details of that project. I used the object data structure for storing Project's details.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -108,18 +108,17 @@ <h2>Tonic</h2>
...
+                    <button class="seeProject-button " id="myBtn">See Project</button>
                 </div>
             </div>
         </div>
         <!-- works section item number 2 -->
-        <div class="works-grid-item item-desktop-2">
+        <div class="works-grid-item item-desktop-2 opacity">
             <div class="desktop">
                 <div class="template-img flex-item-width">
                     <img src="img/Snapshoot Portfolio.png" alt="">
```
Comment (Shisui6) [inline index.html]: According to the project requirements, the projects section should be created dynamically using the information from your projects array. This means that you should completely remove the projects section from the HTML file and create it with javascript when the page loads using your projects array. Please update your project to implement this feature🙏

image

File: index.html [hunk 2]
```diff
@@ -108,18 +108,17 @@ <h2>Tonic</h2>
                     <ul class="ul-list">
                         <button class="buttons"> <li>HTML</li></button>
                         <button class="buttons"><li>CSS</li></button>
```
Comment (Shisui6) [inline index.html]: Nice work on your project cards✨. But to make them look more like the figma design, let's adjust a few things:

Your design:
image

Figma design:
image

A few things to note:
- The background color used in the figma design for the modal is closer to `#d7dae1`. Please update your design to use the same
- Kindly make sure all the content of the modal is left aligned, especially the paragraph text
- Kindly update the background color of the technology list items to be `#ebebff`
- Please update your icons for the see live and see source buttons to be the same as the figma design. You can try exporting the images from figma and styling them to fit your project

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @Salarhabibi-amir, 

# Requires Changes 🍀
You have done a great job so far!👏👏
You still need to work on a few issues before moving on to the next project, but you're almost there🪲!

## Highlights🎯
- Descriptive PR title and summary ✅
- Follows Github flow ✅
- Good commit messages ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🪶
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @Salarhabibi-amir, @Roathy, and @NitBravoA92 ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed GitHub Flow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**